_posts/2024-03-11-release-0_5_0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Kroxylicious release 0.5.0"
+date:   2024-03-10 00:00:00 +0000
+author: "Robert Young"
+author_url: "https://www.github.com/robobario"
+categories:  [releases, kroxylicious, record-encryption]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.5.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.5.0).
+
+### Record Encryption
+
+This release was focused on refining [Record Encryption](/use-cases) to be ready for experimentation in secure environments, adding:
```

Comment:
Grr annoying we can't link directly to the use-case it self.

Reply:
Sure you can, it would be `/use-cases/#encryption-at-rest`, i.e.
```suggestion
This release was focused on refining [Record Encryption](/use-cases/#encryption-at-rest) to be ready for experimentation in secure environments, adding:
```

Reply:
I didn't think we could link directly to catalog entries, is that only true for tabbed ones?